exporter/datadogexporter/config.go

```diff
@@ -40,6 +41,8 @@ var (
 const (
 	// DefaultSite is the default site of the Datadog intake to send data to
 	DefaultSite = "datadoghq.com"
+	// DefaultSumToRateInterval specifies the interval to use when converting a sum to a rate.
+	DefaultSumToRateInterval = 10 * time.Second
```

Comment:
Metrics in OTLP need not be sent at a regular interval, and even if metrics were being sent at a regular interval, this interval may not be a whole number of seconds, which is what our intake currently accepts. Therefore, I am not comfortable accepting this as it is since we are making a transformation that may not make sense semantically.